spyderlib/config.py

```diff
@@ -528,7 +528,7 @@ def is_ubuntu():
               'editor/open file': "Ctrl+O",
               'editor/save file': "Ctrl+S",
               'editor/save all': "Ctrl+Shift+S",
-              'editor/print': "Ctrl+P",
+              'editor/print': "Ctrl+E",

```

Comment:
Please remove this shortcut, don't reassign it. I don't think it's going to be missed.